src/staking-v3/components/leaderboard/LeaderboardVote.vue

```diff
@@ -26,50 +38,54 @@ export default defineComponent({
   setup() {
     const { sortedDapps, isLeaderboardEmpty } = useLeaderboard();
 
-    return { sortedDapps, isLeaderboardEmpty };
+    const { navigateDappPage } = useDappStakingNavigation();
+
+    return { sortedDapps, isLeaderboardEmpty, navigateDappPage };
   },
 });
 </script>
 
 <style lang="scss" scoped>
-@use 'src/css/quasar.variables.scss';
-.wrapper {
-  background-color: $gray-1;
-  border-radius: 16px;
-  padding: 16px;
-  width: 100%;
-}
-.title {
-  font-size: 32px;
-  font-style: normal;
-  font-weight: 700;
-  line-height: normal;
-  margin-bottom: 20px;
-}
-.dapps--container {
-  display: flex;
-  flex-flow: column wrap;
-}
+@import './styles/leaderboard.scss';
+@import './styles/tier.scss';
+// @use 'src/css/quasar.variables.scss';
```

Comment:
Delete commented styles.

Reply:
@bobo-k2 I've deleted them in [8947a54](https://github.com/AstarNetwork/astar-apps/pull/1078/commits/8947a540b381f8e47297d76349ba06c603d443a6)